site/content/3.12/develop/http-api/queries/aql-query-plan-cache.md

```diff
@@ -0,0 +1,143 @@
+---
+title: HTTP interface for the query plan cache
+menuTitle: AQL query plan cache
+weight: 9
+description: >-
+  The query plan cache HTTP API lets you list the AQL query execution plans that
+  are in the cache as well as clear the cache
+---
+## List the entries of the AQL query plan cache
+
+```openapi
+paths:
+  /_db/{database-name}/_api/query-plan-cache:
+    get:
+      operationId: listQueryCachePlans
+      description: |
+        Returns an array containing the AQL query execution plans currently stored
+        in the cache of the selected database.
+      parameters:
+        - name: database-name
+          in: path
+          required: true
+          example: _system
+          description: |
+            The name of the database.
+          schema:
+            type: string
+      responses:
+        '200':
+          description: |
+            The list of cached query plans.
+          content:
+            application/json:
+              schema:
+                description: |
+                  The entries of the query plan cache.
+                type: array
+                items:
+                  description: |
+                    The properties of a cache entry.
+                  type: object
+                  required:
+                    - hash
+                    - query
+                    - queryHash
+                    - bindVars
+                    - fullCount
+                    - dataSources
+                    - created
+                    - numUsed
+                    - memoryUsage
+                  properties:
+                    hash:
+                      description: |
+                        The plan cache key.
+                      type: string
+                    query:
+                      description: |
+                        The query string.
+                      type: string
+                    queryHash:
+                      description: |
+                        The hash value of the query string.
+                      type: integer
+                    bindVars:
+                      description: |
+                        A subset of the original bind parameters with only the
+                        collection bind parameters (e.g. `@@coll`). They need to
+                        have the same names and values for utilizing a cached plan.
+                      type: object
+                    fullCount:
+                      description: |
+                        The value of the `fullCount` query option in the
+                        original query.
```

Comment:
Maybe add that this option causes different execution plans (in general, independent of the plan cache)?